### ISSUE TITLE
Progress bar not working correctly

### ISSUE THREAD

**Issue by antonstsk** [MikaAK/s3-plugin-webpack#137]
# Issue Details
MacOS Catalina 10.15.6
Node v14.4.0
Plugin v1.0.3

Progress bar not only not showing the right upload status its also printing a lot of empty new lines.
![image](https://user-images.githubusercontent.com/33395021/88525429-df19b900-cffa-11ea-8e5c-e83c8e008bd0.png)

Any clue why it's happening?

**Comment by patcon** [comment]
This is the package that's used for progress bar, so answer might lie there: https://www.npmjs.com/package/progress

**Comment by patcon** [comment]
Also, seems that it perhaps hasn't worked in a loooong time :) https://github.com/MikaAK/s3-plugin-webpack/issues/80#issue-243943266

**Comment by MikaAK** [comment]
Yeah it's been bugged for a long time, not sure how to fix it though since by code it still looks right 🤔 If you have any ideas it's only 20LOC https://github.com/MikaAK/s3-plugin-webpack/blob/master/src/s3_plugin.js#L224-L243
